main/pom.xml

```diff
@@ -9,7 +9,6 @@
         <version>1.51.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.github.hazendaz.jmockit</groupId>
```

Comment:
I'll add this back, its a strong preference of mine to be strict in poms of the GAV.  I know m2e in eclipse is all messed up since it has warnings even if set to not warn in recent releases.  Won't bother you with that here, I'll take care of no issues.